packages/app-mobile/root.tsx

```diff
@@ -764,6 +769,8 @@ class AppComponent extends React.Component {
 
 		await setupNotifications(this.props.dispatch);
 
+		await updateRecentsWidget();
```

Comment:
OK, this sounds like a good approach. I also like the debounce mechanism you mentioned, I'll implement that too.

Reply:
Added this in the latest commit.

One bug I noticed: if the user modifies a note and then immediately goes to the home screen, the update won't fire (because the app is no longer in the foreground when the timeout expires), at least not until the app is brought back to the foreground. The only solution that comes to mind is to use something like [headless JS](https://reactnative.dev/docs/headless-js-android), but this honestly sounds like an overkill.